server.js

```diff
@@ -37,6 +37,13 @@ const passport = require("./passportAuthentication");
 app.use(passport.initialize());
 app.use(passport.session());
 
+// socket.io
+var server = require('http').Server(app);
+server.listen(process.env.CHAT_PORT,process.env.CHAT_HOST);
```

Comment:
The host and port for the chat are a piece of information used both the server/ here, and by the client to reach it. Shouldn't we define them in one place that would be used by both sides?